packages/grafana-ui/src/components/Table/ImageCell.tsx

```diff
@@ -27,13 +23,16 @@ export const ImageCell = (props: TableCellProps) => {
           links={() => getCellLinks(field, row) || []}
         >
           {(api) => {
-            const img = <img src={displayValue.text} className={tableStyles.imageCell} alt="" />;
+            const img = (
+              <img
+                style={{ height: tableStyles.cellHeight - DATALINKS_HEIGHT_OFFSET, width: 'auto' }}
+                src={displayValue.text}
+                className={tableStyles.imageCell}
+                alt=""
+              />
+            );
             if (api.openMenu) {
-              return (
-                <Button className={cx(clearButtonStyle)} onClick={api.openMenu}>
-                  {img}
-                </Button>
-              );
+              return <a onClick={api.openMenu}>{img}</a>;
```

Comment:
To address lint issues I think we can just add:
```suggestion
              return <a onClick={api.openMenu} role="button" tabIndex={0}>{img}</a>;
```